### PR TITLE
changed word Tokenizer

### PR DESCRIPTION
**Pull request by JanviRaina**
I made changes in the function and ran it separately and it is working fine. Let me know if this helps.

### CODE REVIEW

File: openodia/_understandData.py [hunk 1]
```diff
@@ -7,20 +8,40 @@
...
+#         str1+=i+' '
+#     token_list = [
+#             token
+#             for token in str1.split()
+#             if token
+#         ]
+#     return token_list
+
+# test_data = "ଭାରତୀୟ ସର୍ବୋଚ୍ଚ ନ୍ୟାୟାଳୟ, ଭାରତର ଉଚ୍ଚତମ ନ୍ୟାୟିକ ଅନୁଷ୍ଠାନ ଅଟେ ।"
+# print(word_tokenizer(test_data))
```
Comment (soumendrak) [inline openodia/_understandData.py:43]: Commented code should not be present in the code.
If past code is needed, can be fetched from `git` commit history.
Keeping commented code reduces readability factor.

File: openodia/_understandData.py [hunk 2]
```diff
@@ -7,20 +8,40 @@
...
     """Tokenizer implementation"""
-
+    
     @classmethod
-    def word_tokenizer(cls, text):
+    def word_tokenizer(cls,text):
         """Split the text into words"""
-        # TODO: Do not remove the punctuation characters
+        punct = regex.compile(r'(\p{Punctuation})')    
+        x=punct.split(test_data)
```
Comment (soumendrak) [inline openodia/_understandData.py:16]: `test_data` should be `text`

Comment (soumendrak) [reply openodia/_understandData.py:16]: Please give the proper variable name depending on what it is and not `x`, `str1` and `i`

File: openodia/_understandData.py [hunk 3]
```diff
@@ -7,20 +8,40 @@
...
+        str1=""
+        for i in x:
+            str1+=i+' '
```
Comment (soumendrak) [inline openodia/_understandData.py:19]: This `for` loop increases additional time complexity to the codebase. 
Is there any better way to do it?

**Review by soumendrak — CHANGES_REQUESTED** [review]
@JanviRaina 
Thanks for the changes. I really appreciate it.
Could you please make the changes requested?
Also, can you please make sure the test cases are passing? Thank you.

**Comment by JanviRaina** [comment]
I have removed the commented portion and replaced the for loop with 'join', have tried to give some better names to the variables.
Thanks for these suggestions.
I am not sure how to test this for all the test cases though. I have only tested it using print statement for this string
("ଭାରତୀୟ ସର୍ବୋଚ୍ଚ ନ୍ୟାୟାଳୟ, ଭାରତର ଉଚ୍ଚତମ ନ୍ୟାୟିକ ଅନୁଷ୍ଠାନ ଅଟେ ।") for which it seemed to have worked fine.

**Comment by soumendrak** [comment]
@JanviRaina run
```shell
black openodia -l 99 -t py39
```
on command prompt to get rid of the linting issue.

**Pull request closed without merging**